src/utils.ts

```diff
@@ -8,6 +8,24 @@
 import { isArray } from '@salesforce/ts-types';
 import { capitalCase } from 'change-case';
 import { JsonObject } from '@salesforce/sf-plugins-core';
+import { Messages } from '@salesforce/core';
+import { AuthInfo } from '@salesforce/core';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-env', 'util');
+
+export type OrgSideEffects = {
+  alias: string;
+  setDefault: boolean;
+  setDefaultDevHub: boolean;
+};
+
+export async function handleSideEffects(authInfo: AuthInfo, sideEffects: OrgSideEffects): Promise<void> {
```

Comment:
This function seems to be duplicated here and in `./src/shared/util.ts`. Can we just have it one place?

```suggestion
export async function handleAuthSideEffects(authInfo: AuthInfo, sideEffects: OrgSideEffects): Promise<void> {
```